ci-operator/config/openshift-knative/serverless-operator/openshift-knative-serverless-operator-main__ocp4.16-lp-interop.yaml

```diff
@@ -106,21 +106,15 @@ tests:
         {
           "failure_rules":
             [
-              {"step": "ipi-*", "failure_type": "all", "classification": "Infrastructure Provisioning - Cluster", "group": {"name": "cluster", "priority": 1}, "jira_additional_labels": ["!default"]},
-              {"step": "install-operators", "failure_type": "all", "classification": "Infrastructure Provisioning - Operators", "jira_additional_labels": ["!default"]},
               {"step": "gather-*", "failure_type": "all", "classification": "Other", "group": {"name": "cluster", "priority": 3}, "jira_additional_labels": ["!default"]},
```

Comment:
I think we can remove this line. I think the rules in the default config that pertain to this step make more sense.